manifests/02-dedicated-admin-operator.ClusterRoleBinding.yaml

```diff
@@ -1,12 +1,12 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: dedicated-admin-operator-admin
+  name: dedicated-admin-operator
 subjects:
 - kind: ServiceAccount
   name: dedicated-admin-operator
   namespace: openshift-dedicated-admin
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: admin
\ No newline at end of file
+  name: dedicated-admin-operator
```

Comment:
The original ClusterRoleBinding is not part of the CSV, bundle, or template anymore because of convoluted logic around what gets put where and that it referenced a ClusterRole that is provided by OCP, not by the operator.  Pulling the RBAC required into a new ClusterRole allows the current logic to work.  Or this change which will technically work as well.  It's a much smaller surface area for the operator.